widgets/accordion/accordion.php

```diff
@@ -226,7 +232,7 @@ public function get_template_variables( $instance, $args ) {
 	}
 	
 	public function render_panel_content( $panel, $instance ) {
-		$content = wp_kses_post( $panel['content_text'] );
```

Comment:
Here again, are you sure we can remove this? Is filtering/sanitization handled by `wpautop`?

Reply:
No. [Sanitization is handled here](https://github.com/siteorigin/so-widgets-bundle/blob/1.17.3/base/inc/fields/tinymce.class.php#L509-L513).

Reply:
Perfect!